bluebottle/assignments/models.py

```diff
@@ -100,12 +98,10 @@ def stats(self):
         contributions = self.contributions.instance_of(Applicant)
 
         stats = contributions.filter(
-            status=ApplicantTransitions.values.succeeded).\
+            status='succeeded').\
```

Comment:
Maybe do an import at the beginning of this method so we can do `ApplicantStateMachine.succeeded.value` here?
Or something like `Applicant.states.succeeded.value` or `self.applicants.__class__.states.succeeded.value`.
Would be nice to skip the literal strings here, right?